streams/src/test/java/org/apache/kafka/streams/kstream/TimeWindowedDeserializerTest.java

```diff
@@ -56,4 +57,23 @@ public void testWindowedValueDeserializerNoArgConstructors() {
         assertNotNull("Inner deserializer should be not null", inner);
         assertTrue("Inner deserializer type should be ByteArrayDeserializer", inner instanceof ByteArrayDeserializer);
     }
+
+    @Test
+    public void setWindowSizeThroughConfigs() {
+        props.put(StreamsConfig.WINDOW_SIZE_MS_CONFIG, "500");
+        final TimeWindowedDeserializer<?> deserializer = new TimeWindowedDeserializer<>();
+        deserializer.configure(props, false);
```

Comment:
Is there some sort of check you can verify here?

Reply:
Ah yeah, good catch